src/index.css

```diff
@@ -1,3 +1,12 @@
+/* Google fonts */
```

Comment:
Great work on this, too! I'll use this to make some changes to `AddItem.jsx` to make sure that the button styling applies there. Also wondering if there was additional styling for the input fields or if those are the default bootstrap styling?

Reply:
@sarahmjenkins the input field style is the default bootstrap style!